RELEASE_NOTES.rst

```diff
@@ -1,3 +1,64 @@
+coala-bears 0.11.0
+==================
+
+This release has been brought to you by 30 contributor contributing over 101
+commits.
+
+**Bug Fixes**
+
+- ``InvalidLinkBear`` now extracts links from markdown correctly.
+  [Issue 1338](https://github.com/coala/coala-bears/issues/1338)
+- ``InvalidLinkBear`` now extracts all the links in a line. Earlier it used to
+  extract only the first match.
+- ``PyCodeStyleBear`` now uses user settings along with default settings.
+  Earlier it used to ignore default settings if user changed one of them.
+- ``PyImpostSortBear`` now recongnizes known third party imports correctly.
```

Comment:
`PyImpostSortBear` -> `PyImportSortBear`

Consider making it more clear: "now correctly supports option 'know_third_party_imports'"